R/geo.R

```diff
@@ -167,4 +167,178 @@ ward_enr_aggs <- function(df) {
 }
 
 
+#' Aggregates assessment data by ward
+#'
+#'
+#' @param list_of_dfs output of \code{fetch_all_parcc}, \code{fetch_parcc},
+#' \code{calculate_agg_parcc_prof}
+#'
+#' @return A data frame of ward aggregations
+#'
+#'
+#' @export
+ward_parcc_aggs <- function(list_of_dfs) {
+   
+   df <- list_of_dfs %>%
+      bind_rows() %>% # convert to df
+      enrich_school_latlong() %>%
+      enrich_school_city_ward()
+   
+   df <- df %>%
+      filter(!is.na(ward) #,
+             #subgroup == "special_education",
+             #grade == 3,
+             #test_name == "ela",
+             #testing_year == 2018,
+             #!is.na(number_of_valid_scale_scores)
+             ) %>%
+      group_by(
+         testing_year,
+         assess_name, test_name,
+         county_id, county_name,
+         district_id, district_name,
+         ward,
+         grade,
+         subgroup, subgroup_type
+      ) %>%
+      summarize( # this should be replaced with parcc_aggregate_calcs()
```

Comment:
did we try doing this with parcc_aggregate_calcs?

Reply:
successfully tried (and tested!) fc950dd353657b9403f0670fef4241de0e5ce251